prisma/schema.prisma

```diff
@@ -149,3 +151,34 @@ model DeliveryLogLine {
   createdAt DateTime    @default(now())
   updatedAt DateTime    @updatedAt
 }
+
+model Conversation {
+  id        Int         @id @default(autoincrement())
+  name      String
+  createdBy Int
+  messages  Message[]
+  createdAt DateTime    @default(now())
+  updatedAt DateTime    @updatedAt
+  users     UsersAndConversations[]
+}
+
+model Message {
+  id                Int           @id @default(autoincrement())
+  createdBy         String
+  content           String
+  userId            Int
+  user              User          @relation(fields: [userId], references: [id])
+  conversationId    Int
+  conversation      Conversation  @relation(fields: [conversationId], references: [id])
+  createdAt         DateTime      @default(now())
+  updatedAt         DateTime      @updatedAt
+}
+
+model UsersAndConversations {
+userId            Int
+user              User  @relation(fields: [userId], references: [id])
+conversationId    Int
+conversation      Conversation  @relation(fields: [conversationId], references: [id])
```

Comment:
If you're handling it explicitly, cascade the delete here too - if a conversation is deleted, so too should records of the conversation members